servicetalk-loadbalancer-experimental-provider/src/main/java/io/servicetalk/loadbalancer/experimental/DefaultLoadBalancerObserver.java

```diff
@@ -44,23 +44,21 @@ public HostObserver hostObserver(Object resolvedAddress) {
     }
 
     @Override
-    public void onServiceDiscoveryEvent(Collection<? extends ServiceDiscovererEvent<?>> events, int oldHostSetSize,
-                                        int newHostSetSize) {
-        LOGGER.debug("{}- onServiceDiscoveryEvent(events: {}, oldHostSetSize: {}, newHostSetSize: {})",
-                lbDescription, events, oldHostSetSize, newHostSetSize);
+    public void onServiceDiscoveryEvent(Collection<? extends ServiceDiscovererEvent<?>> events) {
+        LOGGER.debug("{}- onServiceDiscoveryEvent(events: {})", lbDescription, events);
```

Comment:
Consider still including the size in logs, easier to read than counting events manually